curator/style.css

```diff
@@ -198,6 +198,14 @@ a:not(
 	text-underline-offset: 0.25rem;
 }
 
+/*
+ * Needed until https://github.com/WordPress/gutenberg/issues/39142 is fixed.
+ */
+ .wp-block-navigation__responsive-container.is-menu-open ul {
+	font-size: var(--wp--preset--font-size--huge);
+	font-weight: 300;
```

Comment:
The design suggests the items are also vertically centered, but I don't think we should add CSS for that here.

Reply:
Ah OK.

It would only be one line of CSS, but I know any additional CSS isn't great...

```
.wp-block-navigation__responsive-container.is-menu-open {
	justify-content: center;
}
```

This could be an option in GB, and it probably fits into the other conversations around adding styling controls to the mobile navigation (e.g. https://github.com/WordPress/gutenberg/issues/39142).

Reply:
> It would only be one line of CSS ...

That would only work for menus that fit completely in the window.  Otherwise you can't scroll to get to what's "above" halfway.

There are ways to do it (I **think** we achieved it with Videomaker eventually) but it's not the simplest of solutions.

Reply:
> That would only work for menus that fit completely in the window. Otherwise you can't scroll to get to what's "above" halfway.

Ahh ok, that's not a great solution then!

If it becomes a problem we can add the CSS then, but it sounds like it's best to leave this for now.